docs/records.md

```diff
@@ -72,8 +72,54 @@ So the example is saying:
 - Europe: gets an "A" record of 111.111.111.4
 - Everyone else gets an "A" record of 111.111.111.5
 
+### Health Checks
+
+Octodns will automatically set up monitors for each IP and check for a 200 response for **https://<ip_address>/_dns**.
+
+These checks can be configured by adding a `healthcheck` configuration to the record:
+
+```yaml
+---
+test:
+  geo:
+    AS:
+      - 1.2.3.4
+    EU:
+      - 2.3.4.5
+  octodns:
+    healthcheck:
+      host: my-host-name
+      path: /dns-health-check
+      port: 443
+      protocol: HTTPS
+```
+
+| Key  | Description | Default |
+|--|--|--|
+| host | FQDN for host header and SNI | - |
+| path | path to check | _dns |
+| port | port to check | 443 |
+| protocol | HTTP/HTTPS | HTTPS |
+
+#### Route53 Healtch Check Options
+
+| Key  | Description | Default |
+|--|--|--|
+| measure_latency | Show latency in AWS console | true |
+
+```yaml
+---
+  octodns:
+    healthcheck:
+      host: my-host-name
+      path: /dns-health-check
+      port: 443
+      protocol: HTTPS
+    route53:
+      healthcheck:
+        measure_latency: false
+```
```

Comment:
Nice to have all this documented out. Thanks.

Reply:
You're welcome!